<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>14</y>
     <width>496</width>
     <height>355</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" rowspan="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>108</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" colspan="3">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="loginBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>欢迎使用航班管理系统</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="AccountEdit">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>35</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>请输入账号</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>98</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QLineEdit" name="PasswordEdit">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>35</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请输入密码</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" rowspan="3">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>128</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="2">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
